pkg/domain/update.go

```diff
@@ -56,9 +56,7 @@ func (c *UpdateCommand) Exec(in io.Reader, out io.Writer) error {
 
 	c.updateInput.ServiceID = d.ServiceID
 	c.updateInput.ServiceVersion = d.ServiceVersion
-
-	// Set original value, and then afterwards check if we can use the flag value.
-	c.updateInput.Comment = &d.Comment
+	c.updateInput.Name = d.Name
```

Comment:
This missing field meant the CLI was broken in `master` for updating a domain.

Reply:
Also there's a difference between (for example) domains and dictionaries in that with the domain file we get the current domain and then update the domain, where as with dictionaries we don't bother getting the dictionary we just update it. This is relevant because it means there's an extra API step for domains where as the dictionary assigns the `--new-name` flag value directly into the update struct.

Reply:
It might be worth revisiting the domain implementation to not bother 'getting' the current domain because we no longer are reassigning any pre-existing values.